homeassistant/components/esphome/binary_sensor.py

```diff
@@ -79,6 +84,37 @@ class EsphomeAssistInProgressBinarySensor(EsphomeAssistEntity, BinarySensorEntit
         translation_key="assist_in_progress",
     )
 
+    async def async_added_to_hass(self) -> None:
+        """Create issue."""
+        await super().async_added_to_hass()
+        if TYPE_CHECKING:
+            assert self.registry_entry is not None
+        ir.async_create_issue(
+            self.hass,
+            DOMAIN,
+            f"assist_in_progress_deprecated_{self.registry_entry.id}",
+            breaks_in_ha_version="2025.3",
+            is_fixable=False,
```

Comment:
Should we add a fix flow to these? As in, we can disable it for the user (removing the need for them to search for it..)

Reply:
The user needs to update any dashboards, automations etc. to use the new entity. But I guess we could make a fix flow still.

Reply:
Update with a fix flow